sql/char_equip_saved.sql

```diff
@@ -0,0 +1,26 @@
+SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
```

Comment:
If you're adding new char sql, then you need to update the char entries in trigger.sql, so this new data will automatically get removed on char deletion

Reply:
Added to trigger.sql, thanks for pointing that out! I was lowkey wondering where the deletion process would take care of it considering the comment in packet_system for char deletion is extremely vague.